flybywiresim/aircraft
# fix: SimVar unit names

Fixes #8761 

## Summary of Changes

Rename instances of "celcius" to "celsius" and "foot per minute" to "feet per minute".

## Screenshots (if necessary)


N/A

## References

N/A




## Additional context

N/A


## Testing instructions


- Load aircraft into sim
- Open dev console and search for "unit" related errors
- Observe no errors :)


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
